src/cobra/core/dictlist.py

```diff
@@ -26,7 +27,7 @@ def __init__(self, *args):
 
         """
         if len(args) > 2:
-            raise TypeError("takes at most 1 argument (%d given)" % len(args))
+            raise TypeError(f"takes at most 1 argument ({len(args):d} given)")
```

Comment:
The `:d` format specification is not strictly necessary, I think, but it also doesn't hurt.